src/GovUk.Education.ExploreEducationStatistics.Admin/Migrations/ContentMigrations/20210310143323_EES1979AddEnglishDevolvedArea.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace GovUk.Education.ExploreEducationStatistics.Admin.Migrations.ContentMigrations
+{
+    public partial class EES1979AddEnglishDevolvedArea : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.UpdateData(
+                table: "ContentBlock",
+                keyColumn: "Id",
+                keyValue: new Guid("5d1e6b67-26d7-4440-9e77-c0de71a9fc21"),
```

Comment:
I don't think you want to do this otherwise it will be attempting to do this in Prod!